salt/config/__init__.py

```diff
@@ -1409,7 +1400,6 @@ def _gather_buffer_space():
     'http_max_body': 100 * 1024 * 1024 * 1024,  # 100GB
     'python2_bin': 'python2',
     'python3_bin': 'python3',
-    'cache': 'localfs',
```

Comment:
K, cool. I figured it was something like that but I got pretty nervous when I saw this. ;] Thanks!